### PR TITLE
Limit text files to be loaded

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: controller/filehandlingcontroller.php [hunk 1]
```diff
@@ -75,23 +81,28 @@ public function load($dir, $filename) {
 		try {
 			if (!empty($filename)) {
 				$path = $dir . '/' . $filename;
-				$filecontents = $this->view->file_get_contents($path);
-				if ($filecontents !== false) {
+				// default of 4MB
+				$maxSize = $this->config->getSystemValue("max_size_text_editor", 4194304);
```
Comment (LukasReschke) [inline controller/filehandlingcontroller.php]: :-1: - please no new config.

Comment (LukasReschke) [inline controller/filehandlingcontroller.php]: We should look into https://github.com/owncloud/files_texteditor/pull/57#discussion_r40809970 instead from my PoV.

Comment (LukasReschke) [inline controller/filehandlingcontroller.php]: Let me come up with something that does not require iconv magic :speak_no_evil:

**Comment by LukasReschke** [comment]
Unit test? - Then :+1:

:see_no_evil:

**Comment by DeepDiver1975** [comment]
@LukasReschke

**Pull request merged**